main.tf

```diff
@@ -143,6 +143,17 @@ module "bastion" {
   ami_id           = var.bastion.ami
   instance_type    = var.bastion.instance_type
   kms_key          = local.kms_key_arn
+  security_group_rules = {
+    bastion_inbound_ssh = {
```

Comment:
why is this a map? isn't a list sufficient?

Reply:
Technically it could be, but I like that a map allows for the key to be descriptive of what the rule does.